From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Levy <tomlevy93@gmail.com>
Date: Sun, 14 Jun 2020 02:56:02 +1200
Subject: backport: Change team.effectiveName to team.name in contest site page

Display names and getEffectiveName() were added in commit ac3ec8570.
---
 webapp/templates/jury/contest_site.html.twig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/webapp/templates/jury/contest_site.html.twig b/webapp/templates/jury/contest_site.html.twig
index 3681ab893..0da789372 100644
--- a/webapp/templates/jury/contest_site.html.twig
+++ b/webapp/templates/jury/contest_site.html.twig
@@ -62,7 +62,7 @@
                                 <a href="{{ path('jury_team', {'teamId': team.teamid}) }}">{{ team.teamid }}</a>
                             </td>
                             <td>
-                                <a href="{{ path('jury_team', {'teamId': team.teamid}) }}">{{ team.effectiveName }}</a>
+                                <a href="{{ path('jury_team', {'teamId': team.teamid}) }}">{{ team.name }}</a>
                             </td>
                         </tr>
                     {% endfor %}
-- 
2.20.1

